data/entities.yaml

```diff
@@ -66,11 +66,19 @@
 - name: mountain
   display:
     attr: snow
-    char: '∧'
+    char: '▲'
   description:
   - A mountain.
   properties: [unwalkable]
 
+- name: mountain tunnel
+  display:
+    attr: snow
+    char: '△'
```

Comment:
```suggestion
    char: '△'
    priority: 11
```

Reply:
I added a check in `drawLoc`, but it does not seem to apply - something is not right yet :confused: